src/APICall.py

```diff
@@ -0,0 +1,35 @@
+import requests
+import os
+import json
+
+key = os.environ['FRED_API_TOKEN_REGULATIONS_GOV']
+
+
+def call(url):
+    result = requests.get(url)
+    if (300 <= result.status_code < 400):
+        raise TemporaryException
+    if(result.status_code == 429): #0 calls remaining
+        raise ApiCountZeroException
+    if(result.status_code == 403): #Invlaid API key
```

Comment:
403 is in between 400 and 600... Could be combine if you are raising the same exception